arm-multilib/CMakeLists.txt

```diff
@@ -169,6 +169,7 @@ foreach(lib_idx RANGE ${lib_count_dec})
                 set(read_ENABLE_COMPILER_RT_TESTS "OFF")
                 set(read_ENABLE_LIBCXX_TESTS "OFF")
             else()
+                set(additional_cmake_args "")
```

Comment:
Would this be better off above the if statement? That way if another conditional is added then it will automatically pick up a default "".

Reply:
Good point - I've changed it so every loop starts with an empty list, and the args get appended which makes more sense.